src/Nethermind/Nethermind.HealthChecks/NodeHealthService.cs

```diff
@@ -110,6 +141,37 @@ public CheckHealthResult CheckHealth()
                    _healthHintService.MaxSecondsIntervalForProducingBlocksHint();
         }
 
+        private bool _previousClCheckResult = true;
+        private DateTime _previousClCheckTime = DateTime.MinValue;
+        private int _lastMethodCallSuccesses;
+
+        private bool CheckClAlive(ICollection<(string Description, string LongDescription)> messages, string methodName)
```

Comment:
I thought that we will abstract this check and if post-merge we will run the node without CL we will print some warnings to user, something with meaning: "Dear user! Don't forget to run your CL" :)